README-b2.md

```diff
@@ -0,0 +1,36 @@
+#B2 Ubuntu Specifics
+This is simply a fork of the official Ubuntu linux repository used to create the images that land on Docker hub. We use this fork to ensure images running on our cluster are built from a known, signed source. The only modifications made support tagging images unique to our registry.
+
+The only branch we care about is `dist-amd64`, and it should be used as the base.
+
+## Updating
+1. Make sure you have added teh upstream repo.
+```
+git remote add upstream git remote add upstream https://github.com/tianon/docker-brew-ubuntu-core.git
+```
+2. Fetch the upstream.
+```
+git fetch upstream
+```
+3. Merge the upstream to `dist-amd64`.
+```
+git checkout dist-amd64
+git merge upstream/dist-amd64
+```
+4. Push the changes to this repo.
+```
+git push
+```
+
+## Building
+A build script has been created to build (as of this writing) LTS verions of Ubuntu 14, 16, and 18, as well as the latest 18 verion. Provide the base URL of the repository as an argument while running.
```

Comment:
what is the difference between LTS version 18 and the latest of 18 version?

Reply:
I think I got it :)  So 18 is base version of 18.04.  And latest is the latest updates version of release 18.

Reply:
@QuangHuyNguyen yep exactly - 18 is 18.04, and latest 18 is 18.10 (at the moment). I'll update with specific versions in the doc.